jans-cedarling/cedarling/src/common/policy_store.rs

```diff
@@ -0,0 +1,160 @@
+/*
+ * This software is available under the Apache-2.0 license.
+ * See https://www.apache.org/licenses/LICENSE-2.0.txt for full text.
+ *
+ * Copyright (c) 2024, Gluu, Inc.
+ */
+
+use std::collections::HashMap;
+
+use super::cedar_schema::CedarSchema;
+
+/// PolicyStoreMap it is a store for `PolicyStore` accessible by key.
+#[derive(Debug, serde::Deserialize)]
+pub(crate) struct PolicyStoreMap {
+    #[serde(flatten)]
+    pub policy_stores: HashMap<String, PolicyStore>,
+}
+
+/// PolicyStore contains all the data the Cedarling needs to verify JWT tokens and evaluate policies
+#[derive(Debug, Clone, serde::Deserialize)]
+pub struct PolicyStore {
+    pub schema: CedarSchema,
+    #[serde(deserialize_with = "deserialize_policy_set::parse")]
+    pub policies: cedar_policy::PolicySet,
+}
+
+// Deserialization to [`cedar_policy::PolicySet`] moved here
+// to be close to structure where it is used.
+// And current module is small so it is OK to store it here.
+mod deserialize_policy_set {
+
+    use std::collections::HashMap;
+
+    use base64::prelude::*;
+    use cedar_policy::PolicyId;
+
+    // we use camel case to show that it is like a constant
+    #[derive(Debug, thiserror::Error)]
+    enum ParsePolicySetMessage {
+        #[error("unable to decode policy_content as base64")]
+        Base64,
+        #[error("unable to decode policy_content to utf8 string")]
+        String,
+        #[error("unable to decode policy_content from human redable format")]
+        HumanRedable,
```

Comment:
Maybe `HumanReadable` instead of `HumanRedable`.
```
        #[error("unable to decode policy_content from human readable format")]
        HumanReadable,
```

Reply:
Thanks!